### PR TITLE
Update to latest beta.

### PR DESCRIPTION
**Pull request by vamega**
Not much to say here. Seems like a trivial change to make.

### CODE REVIEW

**Comment by dopplershift** [comment]
Can you drop the post part of the version as well ( so just `3.0.0b3`)? Beta 3 doesn't have any post releases, at least yet.

**Comment by jakirkham** [comment]
It is fortunate that they haven't too as there would be a few more steps if that was the case.

We do something a little strange (but not that strange) by including the C++ built protobuf with `protoc` and the Python package too. So, it is different from the `defaults` version, which is only the Python portion. The post-releases were added for Python only, but not for the C++ part. So, we would need to add those as patches if that was the case.

**Comment by vamega** [comment]
Done.

**Comment by vamega** [comment]
Should I squash the commits before the merge?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,10 +8,10 @@ package:
 source:
     fn: protobuf_{{ version }}.tar.gz
     url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
-    md5: e7f2602baffcbc27fb607de659cfbab6
+    md5: 89afd3855f2d4782e59c09e07d9efa67
 
 build:
-    number: 3
+    number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: This should be `0`, right?

**Comment by jakirkham** [comment]
Sure. Go ahead and squash. Just make sure the build number is `0`. Thanks for the addition.

**Comment by vamega** [comment]
What are we waiting on before we merge?

**Comment by jakirkham** [comment]
Nothing that I know of. Just lost in the craziness of conda-forge. Sorry about that.

@dopplershift, are you ok with this now?

**Comment by dopplershift** [comment]
Yeah, sorry--just got lost in the merge. It should be better now that GitHub let's us know when new pushes have been made to PRs.

**Comment by jakirkham** [comment]
Thanks @dopplershift.

> It should be better now that GitHub let's us know when new pushes have been made to PRs.

😆 If that helps you great, for me it is just more notifications to be buried in.

**Pull request merged**